src/main/java/Duke.java

```diff
@@ -1,10 +1,105 @@
+import java.util.Iterator;
+import java.util.Scanner;
+import java.util.Vector;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        SystemMessages.welcomeMessage();
+
+        boolean breakCheck = false;
+        String input,firstWord;
+        Vector<Task> dataStorage = new Vector<>();
+        int counter = 1;
+        Scanner in = new Scanner(System.in);
+
+        while(!breakCheck) {
+            input = in.nextLine().trim();
+            firstWord = input.split(" ",2)[0];
+
+            switch (firstWord.toLowerCase()) {
+                case "bye":
```

Comment:
I think the case statements should be at the same indentation as the switch word